packages/mobile/src/screens/collection-screen/CollectionScreenDetailsTile.tsx

```diff
@@ -98,7 +98,9 @@ export const CollectionScreenDetailsTile = ({
     // Need to refetch the collection after resetting
     // Will pull from cache if it exists
     // TODO: fix this for smart collections
```

Comment:
No longer TODO?

Reply:
Still todo, we don't support offline smart collections yet. Will need to account for this when we do